File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int hash[] = new int[256];
        int freq[] = new int[256];
        for(char c : s.toCharArray()) hash[c]++;
        for(char c : t.toCharArray()) freq[c]++;

        for(int i = 0; i < 256; i++){
            if(hash[i] != freq[i]) return false;
        }
        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int hash[256] = {0};
        int freq[256] = {0};
        for(char c : s){
            hash[c]++;
        }
        for(char c : t){
            freq[c]++;
        }
        for(int i = 0; i < 256; i++){
            if(hash[i] != freq[i]) return false;
        }
        return true;
    }
};